File: LibraryManagementSystem/Backend/LibraryManagementSystem/LibraryManagementSystem.Entities/Dtos/University/StudentDto.cs
namespace LibraryManagementSystem.Entities.Dtos.University
{
    public class StudentDto
    {
        public int StudentId { get; set; }
        public string? FirstName { get; set; }
        public string? SecondName { get; set; }
        public string? FirstLastname { get; set; }
        public string? SecondLastname { get; set; }
        public string? Carnet { get; set; }
        public string? PhoneNumber { get; set; }
        public char Sex { get; set; }
        public string? Email { get; set; }
        public string? Shift { get; set; }
        public short BorrowedBooks { get; set; }
        public bool HasBorrowedMonograph { get; set; }
        public decimal Fine { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime ModifiedOn { get; set; }
        // Career
        public CareerDto? Career { get; set; }
    }
}

File: LibraryManagementSystem/Frontend/library-management-system-web/src/app/entities/dtos/university/student-dto.ts
import { CareerDto } from "./career-dto";

export interface StudentDto {
    studentId: number;
    firstName: string | null;
    secondName: string | null;
    firstLastname: string | null;
    secondLastname: string | null;
    carnet: string | null;
    phoneNumber: string | null;
    sex: string;
    email: string | null;
    shift: string | null;
    borrowedBooks: number;
    hasBorrowedMonograph: boolean;
    fine: number;
    createdOn: Date;
    modifiedOn: Date;
    career: CareerDto | null;
}